sycl/include/sycl/ext/oneapi/backend/level_zero.hpp

```diff
@@ -83,7 +87,8 @@ T make(const context &Context,
        typename sycl::detail::interop<backend::ext_oneapi_level_zero, T>::type
            Interop,
        ownership Ownership = ownership::transfer) {
-  return make_queue(Context, reinterpret_cast<pi_native_handle>(Interop),
+  auto CommandQ = std::get_if<ze_command_queue_handle_t>(&Interop);
```

Comment:
Could you please clarify what happens if `std::get_if` returns `nullptr` here?

Reply:
The code you see works if the deprecated make<queue> is _recompiled_. However, this API is no longer documented so there would be no point in making it work in recompiled code.

The version of the code up for review now would make it work with existing binaries. (Currently make<queue> is broken because the plugin requires a device pointer and none is supplied. This fixes that bug.)